server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/core/Capability.scala

```diff
@@ -317,7 +317,7 @@ case object DelegationCapabilityFactory extends CapabilityFactory {
 }
 
 final case class SharesCapabilityProperties() extends CapabilityProperties {
-  override def jsonify(): JsObject = Json.obj()
+  override def jsonify(): JsObject = Json.obj("subaddressingSupported" -> true)
```

Comment:
No impact on tests

Reply:
i just wrote one, i am checking that everything is fine and i push

Reply:
It should impact existing session integration tests, no need to write new test IMO